sql/27_stats_area_fence.up.sql

```diff
@@ -0,0 +1,27 @@
+ALTER TABLE `raid_stats`
```

Comment:
I think we can combine the sql query from the other branch to this.. because you already modify a lot.
https://github.com/UnownHash/Golbat/pull/219

Reply:
Done. Also moved area and fence next to date for pokemon_shiny_stats & pokemon_stats, so all tables follow the same format.